cocos/platform/android/jni/JniHelper.h

```diff
@@ -69,6 +69,11 @@ class CC_DLL JniHelper
     static jobject classloader;
     static std::function<void()> classloaderCallback;
 
+
+    /**
+    @brief Call of Java static void method
+    @report error if the condition is not satisfied 
```

Comment:
Which condition? Better to say: In case if no such method will log error

Reply:
I am sorry that I am not good at English. 
Everything you mentioned has been revised. :)